### ISSUE TITLE
Feature: Melee bow attacks when quiver exhausted (?)

### ISSUE THREAD

**Issue by al0fdf** [opennox/opennox#733]
Currently, when exhausting your supply of quivers, Jack shoots from the empty bow, and the message is displayed "No quivers left to equip". This threw me off in the Temple of Ix when I ran out of quivers mid-fight, and was trying to attack a mimic. I remember when playing the game before, that if there were no quivers left, Jack swung the bow like a staff, which still allowed you to defend yourself.

**Comment by angrykirc** [comment]
This is actually not true, there is no melee attack animation for bow
If Jack has no arrows he is able to shoot 'weak ghost arrows' in Nox Quest, in other game modes you still need quivers to shoot

**Comment by al0fdf** [comment]
I could've sworn that there *was* a melee attack animation for the bow, but I've checked the earlier versions of Nox I can find, and all of them display the "No quivers to equip" message. I'm so confused right now.

**Comment by KILAH4716** [comment]
For as long as I've been aware, Bows required Quivers to be useful. It's why shops always offer some kind of Quiver, there are PLENTY to be found throughout Nox, especially in the Land of the Dead, and why in MP, Quivers always spawn next to Bows.

In Quests, however, it's different. They fire a weaker Quiver that only deals 10 damage if you have no Quivers in your inventory. Maybe that's what you're thinking of?

Otherwise, that's Skyrim mechanics, not Nox mechanics.

**Comment by al0fdf** [comment]
> In Quests, however, it's different. They fire a weaker Quiver that only deals 10 damage if you have no Quivers in your inventory. Maybe that's what you're thinking of?

No, I haven't played quest in a while. I must be imagining the fact that bows could be used as melee when quivers are exhausted.
